tests/security/test_sql_injection_prevention.py

```
@@ -238,31 +240,80 @@ async def test_safe_schema_migration(self, backend: SQLiteBackend):
+            async def close(self):
+                return None
+
+        fake_conn = _FakeConn()
+
+        async def _fake_create_connection():
+            return fake_conn
+
+        async_mock_create = AsyncMock(side_effect=_fake_create_connection)
+        async_mock_init = AsyncMock()
+
+        with patch.object(backend, "_create_connection", new=async_mock_create) as mock_conn_factory, \
```

Ruff (0.14.5)

Local variable `mock_conn_factory` is assigned to but never used

Remove assignment to unused variable `mock_conn_factory`

(F841)